packages/gensx/src/index.ts

```diff
@@ -94,12 +101,51 @@ export async function runCLI() {
       .option("-i, --input <input>", "Input to pass to the workflow")
       .option("--no-wait", "Do not wait for the workflow to finish")
       .option("-p, --project <name>", "Project name to run the workflow in")
+      .option("--environment <name>", "Environment name to run the workflow in")
       .option(
         "-o, --output <file>",
         "Output file to write the workflow result to",
         undefined,
       )
       .action(runWorkflow);
+
+    // Environment management commands
+    const environmentCommand = program
+      .command("environment")
```

Comment:
Starting to lean towards making these `env` instead of environment. Feels like we'd then need to change how we specify env vars though (maybe -v and --envVar) 

Curious your thoughts @jmoseley

Reply:
I'm ok with shorter. Definitely can't use `-v` (that is basically always "verbose"). What's wrong with `-e`?

Reply:
Yeah, I think `--env` would be an upgrade here. `-e` is used for exporting environment variables, no?

Reply:
Currently both `-e` and  `--env` are used for exporting environment variables. That's my dilemna here:
- if we change `gensx environment` to `gensx env` these commands have a different meaning for env than the deploy command does
- then for other commands (deploy, run, start), because we use both `-e` and  `--env` for environment variables in the deploy command, there's no good shorthand for `--environment`

Reply:
Well easiest solution might be to change it to a different name. We could replace environment with `stack` and `--stack` and `-s`. But making that change would be painful at this point. 

@jmoseley any thoughts?

Reply:
updated environment to be -e and --env and environment variables to be -ev --env-var